src/sqs/branch.test.ts

```diff
@@ -120,9 +121,10 @@ describe("Branch Webhook", () => {
 		it("should not update Jira issue if there are no issue Keys in the branch name", async () => {
 			const getLastCommit = jest.fn();
 
-			await expect(app.receive(branchNoIssues as any)).toResolve();
+			await expect(app.receive(branchNoIssues)).toResolve();
 			expect(getLastCommit).not.toBeCalled();
 
+			// eslint-disable-next-line @typescript-eslint/require-await
 			await waitUntil(async () => {
```

Comment:
/nit ughh waitUntil :shakesfist: we can get around this suppression with the old promise.all but I'm not sure if thats worse or better lol. Looks like eslint gets in a tiz when callbacks come into play... so it feels like a valid suppress?



```
			await waitUntil(async () => {
				await Promise.all([
					expect(githubNock).toBeDone(),
					expect(jiraNock).toBeDone()
				]);
```